src/tools/blast/blastcommandline.jl

```diff
@@ -0,0 +1,24 @@
+# wrapper for BLAST+ command line functions
+
+using LightXML
```

Comment:
Good to know, though upon further reflection I may try to grab the command-line output rather than write and parse the xml.